UnitsNet.Tests/CustomCode/DensityTests.cs

```diff
@@ -89,6 +89,28 @@ public class DensityTests : DensityTestsBase
 
         protected override double KilogramsPerLiterInOneKilogramPerCubicMeter => 1e-3;
 
+        protected override double TonnesPerCubicFootInOneKilogramPerCubicMeter => 2.8316846591999996e-05;
+
+        protected override double TonnesPerCubicInchInOneKilogramPerCubicMeter => 1.6387063999999997e-08;
```

Comment:
[Verified](https://www.convertunits.com/from/tonnes+per+cubic+inch/to/kilograms+per+cubic+meter)